applications/schedview-snapshot/templates/deployment.yaml

```diff
@@ -77,3 +82,7 @@ spec:
           emptyDir: {}
         - name: slashdatcache
           emptyDir: {}
+        - name: schedview-snapshot-secrets
+          secret:
+            secretName: {{ include "schedview-snapshot.fullname" . }}
```

Comment:
Here, for example, this should just be "schedview-snapshot" instead of this more complex include template.

Reply:
Change made in commit ca02a336.